src/mgr/DaemonServer.cc

```diff
@@ -2418,7 +2503,8 @@ bool DaemonServer::_handle_command(
 	  "\"key\": \"device/" + devid + "\""
 	  "}";
       }
-      auto on_finish = new ReplyOnFinish(cmdctx);
+      op->mark_start_mon_command();
```

Comment:
@rzarzynski @linuxkidd Introduced 2 more mgr op events if op is related to mon command. The op is getting passed to the async callback function `ReplyOnFinish` to mark mon command completion and to help in diagnosis if op is getting stuck on mon side.

Reply:
:+1: